### DISCUSSION TITLE
Mesh does not appear to be applied

### DISCUSSION THREAD

**Discussion by ianthepez** [mriscoc/Ender3V2S1#109] [Q&A] [answered]
I have a mesh created, and use m420 S1 Z2 in my gcode.
The terminal in octoprint shows:
Send: N14 M420 S1 Z2*26
Recv: echo:Bed Leveling ON
Recv: echo:Fade Height 2.00

However, when printing I don't believe it is being applied.  Visually I can't tell for sure, but I also don't see the Z-offset on screen changing, which I was under the impression it would based on this text "If the mesh level compensation is enabled while printing you can see the color change in the z-offset icon:"

Any thoughts or other ways to confirm if the mesh is being applied?

Edit:  After VERY careful examination I can now see that the color of the icon itself slightly changes...I was expecting something a bit more drastic apparently.

**Comment by mriscoc** [comment, marked as answer]
The Z-offset is the physical distance between the z home detected by the probe and the tip of the nozzle. It does not change with the auto leveling.

**Reply by ianthepez** [reply]
Right, I was just expecting some indication of the Z position changing based on the mesh.  The note about the z-offset changing color just led me down the wrong path of expecting it to do something else.
